scripts/testing/update_iso

```diff
@@ -23,9 +23,12 @@ PROJECT_DIR = os.path.dirname(
 # Relative path to the ISO folder within the project
 ISO_FOLDER = os.path.join(PROJECT_DIR, "result", "iso")
 
+# Name of the file storing git revision of the anaconda repository
+REVISION_NAME = "iso.git_rev"
```

Comment:
I would prefer this to be called `REVISION_FILE_NAME` so that it is clear we are talking about a name of a file, not name of a revision.